bench/meson.build

```diff
@@ -0,0 +1,56 @@
+# incdir = include_directories('../include')
+
+if (CLI11_dep.found())
+srcs = ['bm_copy.cc']
+executable('bm_nbt_copy', 
+    srcs, 
+    # include_directories : incdir, 
+    link_language : 'cpp',
```

Comment:
I'm guessing you are overriding to cpp because you don't want to link with the cuda linker?

Reply:
yes, it was causing issues with the CUDA version I had at the time this was implemented.  Not sure if newer versions will link better.

Reply:
Newer versions of Meson don't change anything here. The cuda linker has a higher priority for autodetecting the "best" linker.

There *is* a change in Meson git master for the cuda linker to avoid manually injecting `-lstdc++` (cuda handles this automatically just like the g++ driver) which may conflict with `-lc++`. But I'm not sure that's relevant here. Also, the change is not yet released.